tests/resources/test_material_run.py

```diff
@@ -57,8 +58,8 @@ def test_register_all(collection, session):
     assert [r.name for r in runs] == [r.name for r in registered]
     assert len(session.calls) == 1
     assert session.calls[0].method == 'PUT'
-    assert session.calls[0].path == 'projects/{}/datasets/{}/material-runs/batch'.format(
-        collection.project_id, collection.dataset_id)
+    assert GEMDResourceCollection(collection.project_id, collection.dataset_id, collection.session)._get_path() \
+           in session.calls[0].path
```

Comment:
Even though you asked for `material_run`, it was a `register_all` so it dispatched to `storables`.